src/NUnitFramework/framework/Api/FrameworkController.cs

```diff
@@ -174,7 +177,15 @@ private void Initialize(string assemblyPath, IDictionary settings)
         /// <summary>
         /// Gets a dictionary of settings for the FrameworkController
         /// </summary>
-        public IDictionary Settings { get; private set; }
+        public IDictionary Settings
+        {
+            get { return (IDictionary)InternalSettings; }
+        }
+
+        /// <summary>
+        /// Gets a dictionary of settings for the FrameworkController
+        /// </summary>
+        internal IDictionary<string, object> InternalSettings { get; private set; }
```

Comment:
@CharliePoole  specifically stated that there were third part developers who were, in fact, using the FrameworkController directly, not as an opaque object.

Now he is saying that is not the case. I am understandably confused by all of this.